src/components/cc-zone/cc-zone.js

```diff
@@ -100,13 +103,40 @@ export class CcZone extends LitElement {
             <cc-img class="infra-logo" src=${getInfraProviderLogoUrl(infra)} text=${infra}></cc-img>
           ` : ''}
         </div>
-        <cc-flex-gap class="tag-list">
-          ${zone.tags.map((t) => html`<span class="tag ${classMap({ skeleton })}">${t}</span>`)}
-        </cc-flex-gap>
+        <div class="tag-list">
+          ${zone.tags.map((tag) => this._renderTag(tag, skeleton))}
+        </div>
       </div>
     `;
   }
 
+  /**
+   * @param {string} tag - the tag to render
+   * @param {boolean} skeleton - display as skeleton or not
+   */
+  _renderTag (tag, skeleton) {
+
+    if (tag.includes(':')) {
+      /**
+       * Most of tags are strings separated by ":" but we need to split them in case
```

Comment:
nitpick: most of the time we do multiple single comments especially on these cases. Maybe we should do the same here :thinking: